src/Rings/mpoly-affine-algebras.jl

```diff
@@ -29,15 +29,6 @@ end
 
 @doc Markdown.doc"""
     hilbert_series(A::MPolyQuo)
-
-Given a graded affine algebra $A$ over a field $K$, return a pair $(n,d)$, say, 
-of univariate polynomials in $t$ with integer coefficients
-such that: If $A = R/I$, where $R$ is a multivariate polynomial ring in $n$
-variables over $K$, with positive integer weights $w_1, \dots, w_n$ assigned to the variables,
-and $I$ is a homogeneous ideal of $R$ when we grade $R$ according to the corresponding 
-weighted degree, then $n/d$ represents the Hilbert series of $A$ as a rational function
-with denominator $d = (1-t^{w_1})\cdots (1-t^{w_n})$. See also 
-`hilbert_series_reduced(A::MPolyQuo)`.
```

Comment:
Are you removing this on purpose? It was only added today by @wdecker . Maybe a merge or rebase went wrong?

Reply:
I will take care of this

Reply:
I can bring it back in. It was probably removed by a rebase

Reply:
@fieker merged this, so does this mean we don't want to add it in?

Reply:
Solved by merge with 365